ShenkinStore/Controllers/ShoppingCartController.cs

```diff
@@ -21,19 +21,20 @@ public class ShoppingCartController : Controller
         public ActionResult Index()
         {
          
+          //  var userID = HttpContext.Session.GetString("UserID");
+
             var userID = HttpContext.Session.GetString("UserID");
             
             
             
-            
             if (userID != null)
             {
                 var cart = ShoppingCart.GetCart(userID.ToString());
 
                 var viewModel = new ShoppingCartViewModel
                 {
                     CartItems = cart.GetCartItems(),
-                    CartTotal = cart.GetTotal()
+                    CartTotal = cart.GetTotal2()
```

Comment:
What's the problem with the first one?

Reply:
there is no problem, I tried something else and forgot to remove after.